railties/lib/rails/generators/rails/plugin/templates/Rakefile

```diff
@@ -25,5 +25,5 @@ load 'rails/tasks/statistics.rake'
 
 <% unless options[:skip_gemspec] -%>
 
-Bundler::GemHelper.install_tasks
+require 'bundler/gem_tasks'
```

Comment:
It also does 
```ruby
require "rake/clean"
CLOBBER.include "pkg"
```
Thats included somewhere up the chain? Or this will be newly done now.